.env.example

```
@@ -0,0 +1,8 @@
+POSTGRES_USER_MOVIE=movie_user
+POSTGRES_PASSWORD_MOVIE=your_movie_db_password
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD_MOVIE key should go before the POSTGRES_USER_MOVIE key

(UnorderedKey)